src/PrestaShopBundle/Form/Admin/Sell/Order/Invoices/InvoiceOptionsType.php

```diff
@@ -79,35 +81,160 @@ public function __construct(
+                    ],
+                    'expanded' => true,
+                    'multistore_configuration_key' => 'PS_INVOICE_YEAR_POS',
+                    'label' => $this->trans('Position of the year date', 'Admin.Orderscustomers.Feature'),
+                ]
+            )
+            ->add(
+                'invoice_number',
+                IntegerType::class,
+                [
+                    'required' => false,
+                    'constraints' => [new PositiveOrZero()],
+                    'multistore_configuration_key' => 'PS_INVOICE_START_NUMBER',
+                    'label' => $this->trans('Invoice number', 'Admin.Orderscustomers.Feature'),
+                    'help' => $this->trans(
+                        'The next invoice will begin with this number, and then increase with each additional invoice. Set to 0 if you want to keep the current number (which is #%number%).',
+                        'Admin.Orderscustomers.Help',
+                        [
+                            '%number%' => $this->nextInvoiceNumber,
+                        ]
+                    ),
+                ]
+            )
+            ->add(
+                'legal_free_text',
+                TranslatableType::class,
+                [
+                    'type' => TextareaType::class,
+                    'multistore_configuration_key' => 'PS_INVOICE_LEGAL_FREE_TEXT',
+                    'label' => $this->trans('Legal free text', 'Admin.Orderscustomers.Feature'),
+                    'help' => $this->trans(
+                        'Use this field to show additional information on the invoice, below the payment methods summary (like specific legal information).',
+                        'Admin.Orderscustomers.Help'
+                    ),
+                ]
+            )
+            ->add(
+                'footer_text',
+                TranslatableType::class,
+                [
+                    'type' => TextType::class,
+                    'multistore_configuration_key' => 'PS_INVOICE_FREE_TEXT',
+                    'label' => $this->trans('Footer text', 'Admin.Orderscustomers.Feature'),
+                    'help' => $this->trans(
+                        'This text will appear at the bottom of the invoice, below your company details.',
+                        'Admin.Orderscustomers.Help'
+                    ),
+                ]
+            )
+            ->add(
+                'invoice_model',
+                ChoiceType::class,
+                [
+                    'choices' => $this->invoiceModelChoiceProvider->getChoices(),
+                    'translation_domain' => false,
+                    'multistore_configuration_key' => 'PS_INVOICE_MODEL',
+                    'label' => $this->trans('Invoice model', 'Admin.Orderscustomers.Feature'),
+                    'help' => $this->trans('Choose an invoice model.', 'Admin.Orderscustomers.Help'),
```

Comment:
This could be removed.

Reply:
You're welcome. 

When the string appears in the front office we should take that into consideration, yes. :-)